OrbitCore/Pdb.h

```diff
@@ -154,7 +154,7 @@ class Pdb {
 
   void Init() {}
 
-  bool LoadPdb(const char* file_name) {
+  bool LoadPdb(const char* /*file_name*/) {
```

Comment:
this is why I was a bit confused that clang was not complaining about this line.